docs/discover/document-explorer.asciidoc

```diff
@@ -97,8 +136,7 @@ image::images/document-table-expanded.png[Expanded view of the document table]
 . In the *Table* view, scan through the fields and their values, or search for a field by name.
 
 . When you find a field of interest,
-click
-image:images/actions-icon.png[three dots icon in table column] in the *Actions* column
+hover your mouse over the *Actions* column
```

Comment:
We could also add info about the new action in column dropdown: it's now possible to edit the field and change its name or formatting (as explained in https://www.elastic.co/guide/en/kibana/master/managing-data-views.html#managing-fields)

<img width="440" alt="Screenshot 2022-09-30 at 09 41 12" src="https://user-images.githubusercontent.com/1415710/193218544-1454e671-0bed-4a49-a47b-56c2abb250de.png">

Reply:
Done!